### PR TITLE
[READY]adds fake tear of reality floortiles to the arcade machine

### PR DESCRIPTION
**Pull request by improvedname**
OH MY GOD TEAR OF REALITY AT X LOCATION

:cl: optional name here
add: Adds fake tear of reality floor tiles to the arcade
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
is arcade machine the next braindamage meme

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -34,22 +34,25 @@
...
+		/obj/item/grenade/chem_grenade/glitter/white			= 1,
 		/obj/item/toy/eightball									= 2,
 		/obj/item/toy/windupToolbox								= 2,
 		/obj/item/toy/clockwork_watch							= 2,
 		/obj/item/toy/toy_dagger								= 2,
 		/obj/item/extendohand/acme								= 1,
 		/obj/item/hot_potato/harmless/toy						= 1,
 		/obj/item/card/emagfake									= 1,
+		/obj/item/clothing/shoes/wheelys						= 2,
+		/obj/item/clothing/shoes/kindleKicks				= 2)
```
Comment (81Denton) [inline code/game/machinery/computer/arcade.dm]: missing `,` at the end of the line

**Comment by ExcessiveUseOfCobblestone** [comment]
Can we keep the amount of immersion-breaking/4th wall line-toeing to rare incidences please?

At the very least make it a rare prize.

**Comment by improvedname** [comment]
nah we have fake space and chasms and tear is usually only real when you have a 9x9 space and if somoene uses this to think its fake and auctall fall into a tear then gg

**Comment by ExcessiveUseOfCobblestone** [comment]
This is a bit of a bleeding complaint from the original boh pr but I don't have a problem with the concept of fake tiles, I have a problem with "rift from reality".

I think the concept of "rift in reality" should be used as sparsely as possible and an arcade machine having an item that blatantly represents another item that breaks the fourth wall just doesn't sit right with me.

**Comment by Iamgoofball** [comment]
This is perfectly okay, the arcade machine prizes are the dedicated metabreakers.

**Comment by Shadowflame909** [comment]
Yeah. Isn't that why all the "fake" syndie items exist cobby. You just gotta let dumb captains call the shuttle because of a "boh bomb" in escape. The same type of guys who gib assistants for pricking them with pens!

**Comment by improvedname** [comment]
beside i want people to see that its a fake tear of reality but then 1 tile is real and then they all fall down xp

**Comment by epochayur** [comment]
nice work improvedname, you finally did a good pr

**Comment by ExcessiveUseOfCobblestone** [comment]
@Iamgoofball @Shadowflame909 

It's not about "fake floor" or that the arcade has metabreakers, it's explicitly about this particular item breaking the fourth wall. I'm simply not a fan of the implicit argument "If the boh has it I should be able to put it into the arcade as such!".

I'd be ok if the item was renamed to not be representing a hole in the game world that reveals it's a game. IE "Matrix Carpet" or "Binary-Design Floortiles".

I'm just afraid this will get used to put in more items that aren't exactly the most immersive "because there are literally tiles in the arcade that represent holes of code in the game world".

**Comment by improvedname** [comment]
we have litteraly a fucking thing that shows the codebase who cares lol have fun exdee

**Comment by ExcessiveUseOfCobblestone** [comment]
> I think the concept of "rift in reality" should be used as sparsely as possible

the BoH arena is one location, which I also had a problem with but oh well it's fairly rare.

Again, don't think "One rare area has non-immersive content so now it's open season boys!" is something I agree with.

**Comment by epochayur** [comment]
who cares?

**Comment by ExcessiveUseOfCobblestone** [comment]
Is that because the content isn't immersion breaking or because our playerbase has shifted away from caring about immersion?

Now mind you this is admin-side and not code-side but I specifically recall admins were politely "asked" internally to tone down immersion-breaking content because people seemed to care. That means either a player or an admin got bothered by it (not me) enough to complain to the powers that be.

I don't exactly care either, but there's clearly an audience in our community that does. I think keeping those items localized to rare instances is a perfectly sane request for people who happened to get bothered by it.

I don't think the singular, probably soon-to-be-unfunny gimmick of "haha rift in dorms!" is worth bothering an albeit small (to my knowledge) group of people.

Again, I think it's a neat texture and could be implemented as "Matrix-design floortiles" without directly referencing haha it's a hole that represents the game backdrop!

**Comment by improvedname** [comment]
There only been 6 upvotes and zero downvotes so 6 players already dont care

**Comment by epochayur** [comment]
>I don't exactly care either, but there's clearly an audience in our community that does

Those people are welcome to come here and make points against the PR, then. Your 5 posts in a 15-post thread arguing that someone, somewhere might get slightly offended at a rare, fake tile arcade prize doesn't mean much when no one seems to care.

**Comment by Metroidlover** [comment]
I also agree with cobby. The tear in reality is supposed to represent the game itself being broken open.
The tears shouldn't even be comprehensible to anything IN the game, much less referable. As far as everything in the game goes, the tears shouldn't exist.

However I do still like arcade machines being metabreakers. If you just renamed them to something that doesn't directly reference the tears themselves, I'd be good with them

**Comment by improvedname** [comment]
Oke fine ill do ot when i get on next

**Comment by ExcessiveUseOfCobblestone** [comment]
w-w-w-wait the binary tiles are cool, just not referencing 4th wall breakers.

don't close the pr :(

**Comment by improvedname** [comment]
The frog sees me as a possible !fun! Threat

**Comment by zaracka** [comment]
This is just a white chasm, right? I feel like people are confusing this with the code-textured singularity. I always figured fake floors worked like chameleon projectors. It actually exists in-game, so a mockery is less immersion breaking than the actual thing. The white "chasm" is sci-fi and supposed to represent bluespace or something. I'm looking forward to when they get better sprites, if ever.

**Comment by Shadowflame909** [comment]
If the white chasm is bluespace, and vampires come from the bluespace. Is all this focus on bluespace stealthily trying to tell us vampire the mode is confirm?

**Comment by Iamgoofball** [comment]
You guys are lame, I'll reopen the PR

**Comment by ExcessiveUseOfCobblestone** [comment]
Having goof PR it will certainly up the odds of it getting merged....

**Comment by improvedname** [comment]
ok

**Comment by improvedname** [comment]
Nice joke

**Comment by Bawhoppen** [comment]
I don't know if these specific ones are fourth-wall breaks or not, but we need to take care to not add any more blatant immersion breaks. The thing that ss13 does the best, and the thing where no other game can compare, is how absolutely insanely immersive of a feeling it can provide. Adding random fourth wall jokes individually won't make or break that feeling, but each one will further erode said feeling, just a little bit more.

**Comment by zaracka** [comment]
I don't think this breaks immersion. I think we should remove anything that breaks immersion. Can you give an example?

**Comment by Bawhoppen** [comment]
Thankfully, I don't think we have all that many things right now. Mainly some brain damages lines, and that wizard code singulo. There's probably lots of small unrealistic stuff that also contributes to a lessened immersive atmosphere.
 I'd have to guess that right now, most immersion breaks come from how players/admins act ingame (meta-knowledge, memes, jokes, etc), but that's really kind of a different issue.

**Comment by improvedname** [comment]
Tbh i prefer fake tiles gettable as it could end up in nice designs honestly even if they are jokes like space/chasms

**Comment by bgobandit** [comment]
I don’t get how this has to be OOC in IC, has no one heard of lovecraftian/cosmic horror

**Comment by Iamgoofball** [comment]
Bawhoppenn bro stop posting about ss13 immersion like the graphics are some skyrim level shit

**Comment by Shadowflame909** [comment]
@GuppyLaxx  The hate for this seems to be a minority.

**Comment by improvedname** [comment]
so much drama of this brb fixing rogue pixel

**Comment by improvedname** [comment]
@IconDiffBot

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -34,22 +34,25 @@
...
 		/obj/item/toy/windupToolbox								= 2,
 		/obj/item/toy/clockwork_watch							= 2,
 		/obj/item/toy/toy_dagger								= 2,
 		/obj/item/extendohand/acme								= 1,
 		/obj/item/hot_potato/harmless/toy						= 1,
 		/obj/item/card/emagfake									= 1,
+		/obj/item/clothing/shoes/wheelys						= 2,
+		/obj/item/clothing/shoes/kindleKicks				= 2,
 		/obj/item/clothing/shoes/wheelys				= 2,
 		/obj/item/clothing/shoes/kindleKicks				= 2,
```
Comment (ninjanomnom) [inline code/game/machinery/computer/arcade.dm]: ?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]
The code is fine but I'm honestly not a fan of this

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 3]
```diff
@@ -204,6 +204,13 @@
 	underlay_appearance.icon_state = "basalt"
 	return TRUE
 
+/turf/open/floor/faketear
+	desc = "A clever illusion designed to look like a snow chasm."
+	smooth = SMOOTH_TRUE | SMOOTH_BORDER | SMOOTH_MORE
+	canSmoothWith = list(/obj/item/stack/tile/faketear)
```
Comment (SpaceManiac) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: I assume you want this to smooth with other turfs and not with the item

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 4]
```diff
@@ -132,6 +132,18 @@
 /obj/item/stack/tile/fakepit/loaded
 	amount = 30
 
+/obj/item/stack/tile/faketear
+	name = "snow pits"
+	singular_name = "snow pit"
+	desc = "A piece of carpet with a forced perspective illusion of a pit, Modeled how a pit would look like in the popular snowdin VR training mission."
```
Comment (SpaceManiac) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: capitalization is wonky

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
I don't love these but w/e

**Pull request closed without merging**